scene/gui/progress_bar.cpp

```diff
@@ -85,6 +123,16 @@ void ProgressBar::_notification(int p_what) {
 	}
 }
 
+void ProgressBar::set_fill_mode(int p_fill) {
+	ERR_FAIL_INDEX(p_fill, 9);
```

Comment:
There are only 4 modes at this moment.
```suggestion
	ERR_FAIL_INDEX(p_fill, 4);
```